.github/workflows/build.yml

```diff
@@ -24,6 +24,10 @@ env:
         ccache --max-size=150M
         ccache --set-config=compression=true
 
+concurrency:
```

Comment:
We usually do it this way: Make a private branch inside your fork and only push working commits to the (separate) PRed branch. Then you can always cancel builds that you don't need anymore here: https://github.com/ph4r05/monero/actions

Reply:
sure, but is something wrong with the proposed change? why extra step of keeping 2 separate branches for one PR?